.nycrc

```diff
@@ -1,7 +1,9 @@
 {
-  "check-coverage": true,
-  "lines": 75,
-  "statements": 75,
-  "functions": 75,
-  "branches": 75
+    "check-coverage": true,
+    "lines": 75,
+    "statements": 80,
+    "functions": 75,
+    "branches": 75,
+    "temp-directory": "coverage",
+    "reporter": ["text", "html"]
```

Comment:
`c8` can use `nyc`'s config which makes it easy to go back to `nyc` once we find a path to get that working.